app/main.cjsx

```diff
@@ -4,12 +4,17 @@ ReactDOM = require 'react-dom'
 useScroll = require 'react-router-scroll/lib/useScroll'
 routes = require './router'
 style = require '../css/main.styl'
+{ sugarClient } = require 'panoptes-client/lib/sugar'
 
 # Redux
 `import { Provider } from 'react-redux';`
 `import configureStore from './redux/store';`
+`import { notify, injectSubjects } from './redux/ducks/interventions';`
 store = configureStore()
 
+sugarClient.on('notification', message => store.dispatch(notify(message)));
```

Comment:
Getting ReferenceError on local and staged branch `message is not defined`?

Reply:
That'll be because this should be coffeescript, not ES6.

Reply:
I've added the missing brackets now.